site/get-started.html

```
@@ -127,9 +130,26 @@ <h2 id="action-adoption">GitHub Action adoption</h2>
-      - uses: misty-step/landmark@v1
+      - name: Skip if default branch has advanced
+        id: bound
+        env:
+          GH_TOKEN: ${{ github.token }}
+          TESTED_SHA: ${{ github.event.workflow_run.head_sha || github.sha }}
+          DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
+        run: |
+          set -euo pipefail
+          head="$(gh api "repos/${GITHUB_REPOSITORY}/commits/${DEFAULT_BRANCH}" --jq .sha)"
+          if [ "${head}" != "${TESTED_SHA}" ]; then
+            echo "stale=true" >> "${GITHUB_OUTPUT}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)